### PR TITLE
Ensure consistent tags in the logger.

### PR DESCRIPTION
**Pull request by gitaumoses4**
## Why
- We need to have consistent TAGS in our logs. 

## How
- Update all the tags to have both the class name and the method name separated by a colon (:)

## Testing
- Run a `Find in folder` from Android Studio and verify all occurrences of `Logger.*`

### CODE REVIEW

File: msal/src/main/java/com/microsoft/identity/client/internal/api/BrokerClientIdRefreshTokenAccessor.java [hunk 1]
```diff
@@ -62,15 +62,15 @@
     public static @Nullable
     String get(@NonNull final Context context,
                @NonNull final String accountObjectId) throws MsalClientException {
-        final String methodName = "getBrokerRefreshToken";
+        final String methodTag = TAG + ":getBrokerRefreshToken";
```
Comment (mohitc1) [inline msal/src/main/java/com/microsoft/identity/client/internal/api/BrokerClientIdRefreshTokenAccessor.java]: Should this simply be kept as ":get", for consistency?

Comment (somalaya) [reply msal/src/main/java/com/microsoft/identity/client/internal/api/BrokerClientIdRefreshTokenAccessor.java]: Or can we change the actual method name to getBrokerRefreshToken?

Comment (gitaumoses4) [reply msal/src/main/java/com/microsoft/identity/client/internal/api/BrokerClientIdRefreshTokenAccessor.java]: That's right. Thanks for this.

Comment (gitaumoses4) [inline msal/src/main/java/com/microsoft/identity/client/internal/api/BrokerClientIdRefreshTokenAccessor.java]:
```suggestion
        final String methodTag = TAG + ":get";
```

Comment (gitaumoses4) [reply msal/src/main/java/com/microsoft/identity/client/internal/api/BrokerClientIdRefreshTokenAccessor.java]: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1612/commits/9dc3e28c7bdd69477c6979bc9cb93023604879f9

**Review by mohitc1 — APPROVED** [review]

File: msal/src/main/java/com/microsoft/identity/client/AuthenticationResult.java [hunk 2]
```diff
@@ -126,17 +126,17 @@ public UUID getCorrelationId() {
 
     @Nullable
     private UUID sanitizeCorrelationId(@Nullable final String correlationId) {
-        final String methodName = "sanitizeCorrelationId";
+        final String methodTag= TAG + ":sanitizeCorrelationId";
```
Comment (iamgusain) [inline msal/src/main/java/com/microsoft/identity/client/AuthenticationResult.java]: nit: spacing

Comment (iamgusain) [reply msal/src/main/java/com/microsoft/identity/client/AuthenticationResult.java]: here and few other places..
**methodTag =**

**Review by iamgusain — APPROVED** [review]

File: msal/src/main/java/com/microsoft/identity/client/AuthenticationResult.java (hunk 2 above)
Comment (gitaumoses4) [inline msal/src/main/java/com/microsoft/identity/client/AuthenticationResult.java]:
```suggestion
        final String methodTag = TAG + ":sanitizeCorrelationId";
```

File: msal/src/main/java/com/microsoft/identity/client/AuthenticationResultAdapter.java [hunk 3]
```diff
@@ -58,9 +58,9 @@ static IAuthenticationResult adapt(@NonNull final ILocalAuthenticationResult loc
     static MsalDeclinedScopeException declinedScopeExceptionFromResult(@NonNull final ILocalAuthenticationResult localAuthenticationResult,
                                                                        @NonNull final List<String> declinedScopes,
                                                                        @NonNull final TokenParameters requestParameters) {
-        final String methodName = ":declinedScopeExceptionFromResult";
+        final String methodTag= TAG + ":declinedScopeExceptionFromResult";
```
Comment (gitaumoses4) [inline msal/src/main/java/com/microsoft/identity/client/AuthenticationResultAdapter.java]:
```suggestion
        final String methodTag = TAG + ":declinedScopeExceptionFromResult";
```

File: msal/src/main/java/com/microsoft/identity/client/MultipleAccountPublicClientApplication.java [hunk 4]
```diff
@@ -218,6 +218,8 @@ public void getAccount(@NonNull final String identifier,
     private void getAccountInternal(@NonNull final String identifier,
                                     @NonNull final GetAccountCallback callback,
                                     @NonNull final String publicApiId) {
+        final String methodTag= TAG + ":getAccountInternal";
```
Comment (gitaumoses4) [inline msal/src/main/java/com/microsoft/identity/client/MultipleAccountPublicClientApplication.java]:
```suggestion
        final String methodTag = TAG + ":getAccountInternal";
```

File: msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java [hunk 5]
```diff
@@ -1058,7 +1058,7 @@ protected PublicClientApplication(@NonNull final PublicClientApplicationConfigur
     }
 
     private void initializeApplication() throws MsalClientException {
-        final String methodName = ":initializeApplication";
+        final String methodTag= TAG + ":initializeApplication";
```
Comment (gitaumoses4) [inline msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java]:
```suggestion
        final String methodTag = TAG + ":initializeApplication";
```

File: msal/src/main/java/com/microsoft/identity/client/AuthenticationResult.java (hunk 2 above)
Comment (gitaumoses4) [reply msal/src/main/java/com/microsoft/identity/client/AuthenticationResult.java]: Here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1612/commits/8c46290fffbcb6a6af9d3ae985e8a327198a686a

**Pull request merged**